AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/actions.js

```diff
@@ -177,6 +198,10 @@ define([
                     this.source().set('params.t ', Date.now());
                     mediaBrowser.reload(true);
                     this.preview().getAdobeModal().trigger('closeModal');
+
+                    if (isLicensed) {
+                        mediaGallery.locate(this.getSavedPath(record.id).savedPreviewPath).click();
```

Comment:
The path is either `record.path` (if not empty string) or `destinationPath`. Do we really need an additional request?

Reply:
Both record.path and destination Path contains only current active path, but in our case we need to path where preview was saved?